processor/src/tasks/addToSearch.ts

```diff
@@ -77,7 +77,7 @@ function getReadmeAsText(readmeHTML) {
  * search index documents accordingly.
  */
 export function continuallySyncDeletions(): Promise<SubscriptionHandle> {
-  return giteaDB.subscribeToRepoDeletions(row => {
+  return giteaDB.subscribeToRepoEvents(giteaDB.DBOperation.Delete, async row => {
```

Comment:
feels like this would read a tiny bit nicer if the enum was just called `Operation`

Reply:
Resolved in 75eba2b1.